index.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+// This is the pattern which validate our form.
+$pattern = "/^[A-Za-z]+$/";
+if ($_SERVER["REQUEST_METHOD"] == "POST") {
+	$fname = trim($_POST["fname"], " ");
+	$lname = trim($_POST["lname"], " ");
+	$fullname = $_POST["fullname"];
+	$warning_message = "*Try to fill this field with Alphabet";
+	// This condition is used if someone explicitely try to change disabled field.	
+	if (!empty(htmlspecialchars($fullname))) {
+		$message0 = "You can't explicitly change the disabled field.";
+	} else {
+		$fullname = $fname . " " . $lname;
+		// Below conditions are checking for if both the input field is Properly validate pattern or not,if first input field is properly validate and second input field is not or viceversa.  
```

Comment:
Could we move this to a different function so that other form can use the same validations?